pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -91,10 +91,7 @@ private static void updateAlias(QueryContext queryContext, BrokerResponseNative
     if (resultTable == null) {
       return;
     }
-    Map<ExpressionContext, String> aliasMap = queryContext.getAliasMap();
-    if (aliasMap.isEmpty()) {
-      return;
-    }
+    List<String> aliasList = queryContext.getAliasList();
```

Comment:
Do we need to check if it is empty? I think it will be empty for PQL query?

Reply:
+1

Reply:
Yes, it's empty for PQL. Added it back to short circuit for PQL.